release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1694,6 +1694,8 @@ In the following tables, features are marked with the following statuses:
 
 * When running a cluster with FIPS enabled, you might receive the following error when running an older version of the OpenShift CLI (`oc`) on a {op-system-base} 9 system: `FIPS mode is enabled, but the required OpenSSL backend is unavailable`. As a workaround, use the `oc` binary provided with the {product-title} cluster. (link:https://issues.redhat.com/browse/OCPBUGS-23386[*OCPBUGS-23386*])
 
+* In {product-version} with IPv6 networking running on {rh-openstack-first} environments, `IngressController` objects configured with `endpointPublishingStrategy.type=LoadBalancerService` will not function correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2263550[*BZ#2263550*], link:https://bugzilla.redhat.com/show_bug.cgi?id=2263552[*BZ#2263552*])
```

Comment:
Looks good. I couldn't figure on how to get a noun after `endpoint...`. I think we are ok though because you have objects configured with. The only thing you could do is maybe say `endpoint...` YAML configuration or maybe install configuration or deployment config.

Reply:
That's great, thanks. I couldn't come up with a solution either! I'll make the change.